### PR TITLE
1D laser acceleration

### PR DESCRIPTION
**Pull request by prkkumar**
This a PR adds a 1D laser acceleration test

### CODE REVIEW

File: Examples/Physics_applications/laser_acceleration/inputs_1d [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+algo.particle_shape = 3
+
+#################################
+############ PLASMA #############
+#################################
+particles.species_names = electrons
+
+electrons.species_type = electron
+electrons.injection_style = "NUniformPerCell"
+electrons.num_particles_per_cell_each_dim = 1
```
Comment (ax3l) [inline Examples/Physics_applications/laser_acceleration/inputs_1d]: Maybe adding a few particles more makes sense for 1D:
```suggestion
electrons.num_particles_per_cell_each_dim = 10
```

File: Examples/Physics_applications/plasma_acceleration/PICMI_inputs_plasma_acceleration_1d.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+constants = picmi.constants
+
+nz = 64
+
+zmin = -200.e-6
+zmax = +200.e-6
+
+moving_window_velocity = [0., 0., constants.c]
+
+number_per_cell_each_dim = [2, 1, 1]
```
Comment (ax3l) [inline Examples/Physics_applications/plasma_acceleration/PICMI_inputs_plasma_acceleration_1d.py]: Does PICMI always expect a len 3 list here?

particles: maybe a few more makes sense per cell, e.g., `10`?
```suggestion
number_per_cell_each_dim = [10]
```

**Review by ax3l — COMMENTED** [review]
LGTM, just tiny questions inline

File: Examples/Physics_applications/laser_acceleration/inputs_1d [hunk 3]
```diff
@@ -0,0 +1,66 @@
+#################################
+####### GENERAL PARAMETERS ######
+#################################
+max_step = 1000
```
Comment (ax3l) [inline Examples/Physics_applications/laser_acceleration/inputs_1d:4]: Can you say how long the test runs with 2 MPI ranks  & 1 OMP thread? Just checking it's short enough for CI.

(Python test below uses 10 steps)

Comment (prkkumar) [reply Examples/Physics_applications/laser_acceleration/inputs_1d:4]: Total time = 0.4985202 with 2 MPI, 1 OMP, 1000 steps

Comment (ax3l) [reply Examples/Physics_applications/laser_acceleration/inputs_1d:4]: Perfect, that's totally fine.

Do you then like to increase the number of steps in the Python test, too?

Comment (prkkumar) [reply Examples/Physics_applications/laser_acceleration/inputs_1d:4]: Sure, how many steps would you recommend?

File: Examples/Physics_applications/plasma_acceleration/PICMI_inputs_plasma_acceleration_1d.py (hunk 2 above)
Comment (prkkumar) [reply Examples/Physics_applications/plasma_acceleration/PICMI_inputs_plasma_acceleration_1d.py]: No, just one number in the list is enough.

File: Examples/Physics_applications/laser_acceleration/inputs_1d (hunk 1 above)
Comment (prkkumar) [reply Examples/Physics_applications/laser_acceleration/inputs_1d]: Agreed!

**Pull request closed without merging**